<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="IZDT-211" ignored="0" name="Scenario 1 Test Cases" tests="9" failures="1" timestamp="26 Jul 2018 07:22:50 GMT" time="6.714" errors="0">
  <testcase name="@AfterMethod getResult" time="0.003" classname="com.core.reports.ExtentReport">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.core.reports.ExtentReport.getResult(ExtentReport.java:50)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:497)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- @AfterMethod getResult -->
  <testcase name="@AfterMethod getResult" time="0.0" classname="com.core.reports.ExtentReport">
    <skipped/>
  </testcase> <!-- @AfterMethod getResult -->
  <testcase name="@AfterMethod getResult" time="0.0" classname="com.core.reports.ExtentReport">
    <skipped/>
  </testcase> <!-- @AfterMethod getResult -->
  <testcase name="@AfterMethod getResult" time="0.0" classname="com.core.reports.ExtentReport">
    <skipped/>
  </testcase> <!-- @AfterMethod getResult -->
  <testcase name="@AfterMethod getResult" time="0.0" classname="com.core.reports.ExtentReport">
    <skipped/>
  </testcase> <!-- @AfterMethod getResult -->
  <testcase name="@AfterMethod getResult" time="0.0" classname="com.core.reports.ExtentReport">
    <skipped/>
  </testcase> <!-- @AfterMethod getResult -->
  <testcase name="@AfterMethod getResult" time="0.0" classname="com.core.reports.ExtentReport">
    <skipped/>
  </testcase> <!-- @AfterMethod getResult -->
  <testcase name="@AfterMethod getResult" time="0.0" classname="com.core.reports.ExtentReport">
    <skipped/>
  </testcase> <!-- @AfterMethod getResult -->
  <testcase name="@AfterMethod getResult" time="0.0" classname="com.core.reports.ExtentReport">
    <skipped/>
  </testcase> <!-- @AfterMethod getResult -->
  <testcase name="testSC_1" time="1.919" classname="com.r1v2.reporting.test.Scenario1Login">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at com.r1v2.reporting.test.Scenario1Login.testSC_1(Scenario1Login.java:41)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:497)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- testSC_1 -->
  <testcase name="testSC_2" time="0.0" classname="com.r1v2.reporting.test.Scenario1Login">
    <skipped/>
  </testcase> <!-- testSC_2 -->
  <testcase name="testSC_3" time="0.0" classname="com.r1v2.reporting.test.Scenario1Login">
    <skipped/>
  </testcase> <!-- testSC_3 -->
  <testcase name="testSC_4" time="0.0" classname="com.r1v2.reporting.test.Scenario1Login">
    <skipped/>
  </testcase> <!-- testSC_4 -->
  <testcase name="testSC_5" time="0.0" classname="com.r1v2.reporting.test.Scenario1Login">
    <skipped/>
  </testcase> <!-- testSC_5 -->
  <testcase name="testSC_6" time="0.0" classname="com.r1v2.reporting.test.Scenario1Login">
    <skipped/>
  </testcase> <!-- testSC_6 -->
  <testcase name="testSC_7" time="0.0" classname="com.r1v2.reporting.test.Scenario1Login">
    <skipped/>
  </testcase> <!-- testSC_7 -->
  <testcase name="testSC_8" time="0.0" classname="com.r1v2.reporting.test.Scenario1Login">
    <skipped/>
  </testcase> <!-- testSC_8 -->
  <testcase name="testSC_9" time="0.0" classname="com.r1v2.reporting.test.Scenario1Login">
    <skipped/>
  </testcase> <!-- testSC_9 -->
</testsuite> <!-- Scenario 1 Test Cases -->
